### ISSUE TITLE
Add `font-display: swap;` to generated css file

### ISSUE THREAD

**Issue by mangelozzi** [majodev/google-webfonts-helper#135]
When performing a Chrome lighthouse audit, the its fails and recommends adding 
`font-display: swap;`

<https://web.dev/font-display/?utm_source=lighthouse&utm_medium=devtools#how-to-avoid-showing-invisible-text>

E.g.:
```css
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: local(''),
       url('/static/fonts/roboto-v29-latin-300.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
       url('/static/fonts/roboto-v29-latin-300.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
}
```

becomes

```css
@font-face {
  font-family: 'Roboto';
  font-style: normal;
  font-weight: 300;
  src: local(''),
       url('/static/fonts/roboto-v29-latin-300.woff2') format('woff2'), /* Chrome 26+, Opera 23+, Firefox 39+ */
       url('/static/fonts/roboto-v29-latin-300.woff') format('woff'); /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
  font-display: swap;
}

```

**Comment by danger89** [comment]
Must have indeed.

**Comment by majodev** [comment]
Fixed within [v1.1.0](https://github.com/majodev/google-webfonts-helper/releases/tag/v1.1.0).
